index.html

```diff
@@ -2384,38 +2384,34 @@ <h3>Processing Errors</h3>
 below.
           </li>
           <li>
-The `code` value MUST be the integer code described in the table below
-(in parentheses, beside the type name).
-          </li>
-          <li>
-The `title` value SHOULD provide a short but specific human-readable [=string=] for
+The `title` value MUST be present and SHOULD provide a short but specific human-readable [=string=] for
 the error.
           </li>
           <li>
-The `detail` value SHOULD provide a longer human-readable [=string=] for the error.
+The `detail` value MUST be present and SHOULD provide a longer human-readable [=string=] for the error.
```

Comment:
This should match what's in the other specs and not add a new MUST: https://w3c.github.io/vc-bitstring-status-list/#processing-errors

```suggestion
The `detail` value SHOULD provide a longer human-readable [=string=] for the error.
```

Reply:
See my comment above

Reply:
See comment here: https://github.com/w3c/vc-data-integrity/pull/327/files#r1912460821